src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -14,7 +14,7 @@ import './RegisterForm.css';
 
 export default function RegisterForm(): JSX.Element {
     const history = useHistory();
-    const user = useUserContext();
+    const { user } = useUserContext();
```

Comment:
`pending` was used for login form. Does it need to be use here too? 

Although, I'm really clear on if `pending` affects the outcome.

Reply:
Mmm yeah we can use `pending` here similar to log in because after a new user goes to create an account we can show a spinner as well.